README.md

```diff
@@ -28,7 +28,7 @@ c = c + d;
 
 After this code executes, what is the value of c?  Please identify the operators in the last line of this program `c = c + d;` and explain what they do.
 
-<!-- Replace this comment with your answer -->
+c = 7, operators are +=
```

Comment:
Just remember that `=` isn't 'equals', like in math, it's 'assign/let', 
i.e. 
Let c be the sum of c and d.